main/src/ca/uwaterloo/flix/language/phase/Lexer.scala

```diff
@@ -200,6 +200,17 @@ object Lexer {
     }
   }
 
+  /**
+   * Peeks the character two before the previous that state was on if available.
```

Comment:
three? Sorry I don't even understand the comment, actually.

Reply:
It was basically what `previousPrevious` did but one more. I just generalized it to work for `n` characters - maybe that will be easier to understand. Alternatively, I could just inline the function and wouldn't need a separate definition